www/modules/custom/borg_mailers/borg_mailers_security/borg_mailers_security.module

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * @file borg_mailers_security.module
+ * Send out Security Anouncement emails to users who have subscribed.
+ *
+ * Users can check a box upon editing there user profile (/user when logged in)
+ * to opt into the Backdrop Security email list.
+ *
+ * Security emails will go out on publishing a security release node of Backdrop
+ * core or any Backdrop Contrib project.
+ */
+
+/**
+ * Implements hook_node_submit().
+ *
+ * @see _borg_mailers_record_mail_sent().
+ * @see _borg_mailers_security_subscribers().
+ * @see _borg_mailers_record_mail_sent().
+ */
+function borg_mailers_security_node_submit($node) {
+  // Save the node right quick so we can get an nid.
+  $node->save();
```

Comment:
It seems that saving the node before the system is ready to save the node would be problematic. Maybe we need to use `hook_node_presave()` or some other hook where we already have a NID...